tools/docker_dev/Dockerfile

```diff
@@ -98,6 +99,14 @@ RUN sed -i 's/^#force_color_prompt=yes/force_color_prompt=yes/' /etc/skel/.bashr
     # Add call to conda init script see https://stackoverflow.com/a/58081608/4413446
     echo 'eval "$(command conda shell.bash hook 2> /dev/null)"' >> /etc/skel/.bashrc 
 
+# -----------------------------------------------------------------------------
+
+# Enable basic vim defaults in ~/.vimrc
+RUN echo "set number" >> ~/.vimrc && \
+    echo "syntax enable" >> ~/.vimrc && \
+    echo "filetype plugin indent on" >> ~/.vimrc && \
+    echo "set colorcolumn=80" >> ~/.vimrc
```

Comment:
those mostly look familiar to me anyway

Reply:
I am happy with these being added. @rgommers  do you have any other configs you think would be useful here?

Reply:
This is probably enough for a generic config. I just browsed my own `.vimrc` and the other stuff I have in there is too specific to put in here.